proposals/0310-PerformInteractionMultipick.md

```diff
@@ -61,7 +61,8 @@ and update the `HMI_API`
 
 ```
 
-On older head units, the app might request a multiselect `performInteraction` but the app would get a response of INVALID_DATA. So the app would have to handle business logic while thats going on
+ On older head units, the app might request a multiselect `performInteraction` but if not available, the head unit will responsd with `INVALID_DATA`. From that point, it would be up to the app on how to handle that logic.
```

Comment:
```suggestion
 On older head units, the app might request a multiselect `performInteraction` but if not available, the head unit will respond with `INVALID_DATA`. From that point, it would be up to the app on how to handle that logic.
```